state/movie/__tests__/selectors.test.js

```diff
@@ -0,0 +1,46 @@
+import {
+  isMovieLoading,
+  movieError,
+  movieDetails
+} from '../selectors';
+
+describe('Movie selectors.', () => {
+  const loading = false;
+  const error = 'Mock error';
+  const state = {
+    movie: {
+      loading,
+      error,
+      id: 1
+    },
+    data: {
+      movies: {
+        1: { id: 1, title: 'First mock title' },
+        2: { id: 2, title: 'Second mock title' }
+      }
+    }
+  };
+
+  it('isMovieLoading() returns loading status', () => {
+    expect(isMovieLoading(state)).toEqual(loading);
```

Comment:
You can use .toBeFalsy()
https://jestjs.io/docs/en/expect#tobefalsy

Reply:
Fixed